docs/faq.md

```diff
@@ -40,7 +40,6 @@ local               doccano_postgres_data
 local               doccano_static_volume
 local               doccano_venv
 local               doccano_www
-❯ docker volume rm doccano_www
-❯ docker volume rm doccano_static_volume
+❯ docker volume rm doccano_node_modules doccano_static_volume doccano_venv doccano_venv
```

Comment:
- `doccano_www` disappears
- `doccano_venv` is duplicated

Reply:
Oh, sorry. Done